metatron-app/metatron-discovery
# add toggle button in map and check multiple geo field on same layer

### Description
1. Added a button to turn on/off data inquiry on the map
2. Added ability to check that multiple geo fields cannot be placed on the same layer

**Related Issue** : 


### How Has This Been Tested?
1. Enter the chart creation screen.
2. After selecting the map chart, try to place 2 geo fields on the same layer.
3. Check for warning messages.
4. Check that the Y/N button is displayed in the upper left corner of the map.
5. When the button is in N state, check if data inquiry is disabled.

#### Need additional checks?

### Types of changes

- [x] New feature (non-breaking change which adds functionality)


### Checklist:


### Additional Context
